lib/sandthorn.rb

```diff
@@ -122,6 +110,44 @@ def event_stores
       def event_store=(store)
         @event_stores = EventStores.new(store)
       end
+
+      def serializer= block
```

Comment:
I really prefer parentheses around method args. Perhaps we should consider using Rubocop?